vizro-core/tests/unit/vizro/actions/_action_loop/test_get_action_loop_components.py

```diff
@@ -141,7 +141,7 @@ def test_no_components(self):
         ],
         indirect=True,
     )
-    def test_all_action_loop_components(  # pylint: disable=too-many-arguments  # noqa: PLR0913
+    def test_all_action_loop_components(  # pylint: disable=too-many-arguments
```

Comment:
Are we still running pylint?

Reply:
It seems like we do - it's still specified under [tool.ruff.lint] as a rule 👍